docs/source/user/subdyn/input_files.rst

```diff
@@ -525,22 +527,26 @@ An example of rigid link properties table is given below
      (-)       (kg/m)    
       12       7850.0
        3       7000.0
-       
-
-
-
-
-
-
-
-
-
-
-
 
+Spring Properties
+~~~~~~~~~~~~~~~~
+Members that are specified as spring elements (**MType=5**), 
+have their properties defined in the spring element properties table. 
+The table lists for each spring property: the property ID (**PropSetID**), the diagonal stiffness 
+coefficients (**K11**, **K22**, **K33**, **K44**, **K55**, **K66**), and the cross-coupling
```

Comment:
The description could be simplified by stating:
`The table lists for each spring property: the property ID (**PropSetID**) and the stiffness coefficients (**K11**, **K12**, **K13**, **K14**, **K15**, **K16**, **K22**, **K23**, **K24**, **K25**, **K26**,  **K33**, **K34**, **K35**, **K36**, **K44**, **K45**, **K46**, **K55**, **K56**, **K66**).`

Like this it would appear in the same order as in the input file.